locales/base/translation.json

```diff
@@ -1928,7 +1928,8 @@
       "myTokens": "My tokens",
       "popular": "Popular",
       "recentlySwapped": "Recently swapped",
-      "network": "{{networkName}} Network"
+      "network": "{{networkName}} Network",
+      "selectNetwork": "Network"
```

Comment:
I think we've been doing version suffixes for new version of the same key, but this works too